src/main/java/org/example/csv2tex/placeholders/schoolspecific/ErfurtSchoolTablePlaceholderReplacer.java

```diff
@@ -63,7 +63,11 @@ public class ErfurtSchoolTablePlaceholderReplacer implements TablePlaceholderRep
     private static final String LEVEL_RED = "\\levelThree";
     private static final String LEVEL_BLUE = "\\levelTwo";
     private static final String LEVEL_GREEN = "\\levelOne";
+    private static final String LEVEL_SEVEN = "\\levelSeven";
+    private static final String LEVEL_EIGHT = "\\levelEight";
+    private static final String LEVEL_NINE = "\\levelNine";
     private static final String LEVEL_None = "\\noLevel";
+    private static final String LEVEL_GB = "\\noLevel";
```

Comment:
GB = noLevel ?

What is "GB" an abbreviation for?

Reply:
geistig behindert ^^

Reply:
I seee. Maybe, we can go with "mental disability" or something, then. 
https://en.wikipedia.org/wiki/Mental_disorder#Disability